bcgov/HOUS-permit-portal
# Add support for admin login with IDIR

## Description

Adds a separate login screen for admins at `/admin` where admins can sign in with their IDIR. Login only works for existing admin accounts. New admins must be invited by another admin.

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Implements [BPHH-1226](https://hous-bssb.atlassian.net/browse/BPHH-1226)

## Steps to QA

1. Navigate to `/admin` and try to login with your IDIR. Login should fail and the user is directed back to the regular login page with a message indicating that self registration for admins is not supported.
2. Login as the seeded super admin via the regular login page (bceid `super_admin` / `P@ssword1`
3. Invite an admin. Open the invitation in an incognito window and login with your IDIR. You should now have access as an admin.
4. Invite another admin. Open the invitation and try to login with the same IDIR. You should be redirected to the regular login page with a message indicating the IDIR is taken. 

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
